IDE/Renesas/e2studio/RX65N/GR-ROSE/test/src/key_data.c

```diff
@@ -90,56 +90,56 @@ const uint32_t              encrypted_user_key_type =
+    0x74, 0x57, 0x83, 0x85, 0xba, 0x21, 0x7a, 0xc3, 0x84, 0xb3, 0x0f, 0x23,
+	0x69, 0x07, 0xf8, 0xcc, 0x1a, 0x66, 0xc0, 0xcd, 0x88, 0x35, 0x13, 0x60,
+    0xd2, 0xe6, 0x97, 0xc3,	0xf0, 0xc1, 0xee, 0xf1, 0x00, 0x56, 0x3e, 0xf2,
+    0x42, 0xc2, 0x58, 0x56, 0xa2, 0x8f, 0xe4, 0x0d,	0x5b, 0xac, 0x59, 0x27,
+    0x35, 0x0a, 0x88, 0xb7, 0x9f, 0x81, 0xcf, 0xf1, 0xfd, 0x11, 0x07, 0x15,
+	0x46, 0x99, 0x9c, 0x41, 0x9d, 0x0c, 0x3d, 0x4d, 0x30, 0x91, 0x16, 0x8a,
+    0xd8, 0xd2, 0xa2, 0x4b,	0xa6, 0x0b, 0x7b, 0x7b, 0xb1, 0x2e, 0x34, 0xb5,
+    0x0c, 0xda, 0x61, 0x6c, 0xb0, 0xb1, 0x3a, 0x70,	0xad, 0xcc, 0xe1, 0x9e,
+    0x48, 0x41, 0xd7, 0x18, 0xb7, 0x9c, 0xb4, 0x38, 0xd3, 0xc3, 0x2b, 0xf0,
+	0x23, 0x08, 0x4e, 0x2a, 0x2e, 0xa1, 0x80, 0x22, 0x24, 0x77, 0x67, 0xab,
+    0x2b, 0x58, 0xc2, 0x51,	0xaf, 0x2c, 0xf2, 0x61, 0x31, 0x6f, 0xce, 0xc8,
+    0x86, 0x96, 0xe7, 0xa7, 0x14, 0x07, 0x65, 0xda,	0xf2, 0xba, 0x83, 0x45,
+    0xe1, 0x5c, 0x51, 0x29, 0xdd, 0x34, 0xc2, 0x37, 0x14, 0x50, 0x89, 0xae,
+	0x95, 0xa7, 0xa7, 0xbb, 0xc4, 0x79, 0xb8, 0xd1, 0xae, 0x1d, 0x48, 0x6d,
+    0x88, 0x0f, 0x41, 0xb1
 };
 const int sizeof_ca_ecc_cert_sig = sizeof(ca_ecc_cert_der_sig);
 
 /* ./ca-cert.der.sign,  */
 const unsigned char ca_cert_der_sig[] =
 {
-	0x0E, 0xC3, 0x9B, 0x77, 0xF8, 0x58, 0x08, 0x9E, 0x5D, 0x1E, 0x03, 0x8D,
-    0x60, 0xD1, 0xF6, 0x3E, 0x3D, 0xFF, 0x89, 0x4C, 0x91, 0x5C, 0x00, 0xEB,
-    0x05, 0xE5, 0x65, 0x62, 0x17, 0xFB, 0xD4, 0x52,	0x69, 0x9D, 0xB8, 0x07,
-    0xAF, 0xA9, 0x4C, 0xA5, 0xB9, 0x8D, 0x52, 0xC0, 0xF3, 0x34, 0x13, 0x67,
-	0x40, 0xAA, 0xE1, 0xA3, 0x9E, 0x5D, 0x0F, 0xCE, 0x87, 0xB0, 0x10, 0xB4,
-    0x79, 0x8F, 0x84, 0x21,	0x81, 0xC2, 0xF9, 0xF7, 0xDB, 0xCB, 0x8F, 0xE4,
-    0x9B, 0xF5, 0x85, 0x9D, 0x11, 0x04, 0xFB, 0xA7,	0xFD, 0x13, 0x6F, 0x02,
-    0xA5, 0xBF, 0xE0, 0x89, 0x62, 0x5E, 0x24, 0x95, 0xF6, 0x01, 0x7D, 0x7F,
-	0xB5, 0xD1, 0xDD, 0xF3, 0x3B, 0xD5, 0x04, 0x54, 0xE1, 0x8E, 0xA8, 0x3D,
-    0x30, 0xB3, 0x35, 0x76,	0xAF, 0xA7, 0x94, 0xD7, 0x59, 0x82, 0x38, 0x2C,
-    0xD6, 0x95, 0x57, 0xD1, 0xD5, 0x62, 0xB1, 0x69,	0x60, 0xCD, 0x3F, 0x7D,
-    0x0E, 0x9F, 0x00, 0x21, 0x04, 0xFE, 0x43, 0xBD, 0x7D, 0x3D, 0xA7, 0x6B,
-	0xC5, 0x82, 0x92, 0xDE, 0xB7, 0xA3, 0xD4, 0x7D, 0x3C, 0x14, 0x46, 0x28,
-    0x50, 0xCA, 0x86, 0x9F,	0x66, 0x4C, 0xB0, 0x46, 0x46, 0x4D, 0x31, 0xD6,
-    0x7B, 0xEC, 0xBA, 0xED, 0xA1, 0xF9, 0x88, 0x68,	0xB9, 0xA9, 0xDA, 0x88,
-    0x63, 0x01, 0x95, 0x5B, 0x78, 0x38, 0x03, 0xD6, 0xDF, 0x86, 0xC4, 0x3E,
-	0x3B, 0xCF, 0xED, 0x8B, 0x2A, 0x41, 0x49, 0x65, 0x3E, 0x2F, 0x45, 0x71,
-    0xD8, 0x0B, 0xF1, 0xF0,	0xC7, 0xB5, 0x2E, 0xBE, 0xF0, 0x71, 0xDE, 0x40,
-    0xB0, 0x54, 0x25, 0xD7, 0x4A, 0x86, 0xF1, 0xB9,	0xF6, 0xAB, 0x07, 0x07,
-    0x21, 0x7C, 0x15, 0x7B, 0x1F, 0xCF, 0xE4, 0x1F, 0x0B, 0xEB, 0x0E, 0x96,
-	0xE5, 0x59, 0x34, 0xC6, 0x4B, 0x1B, 0xF6, 0xC7, 0x6C, 0x4C, 0x16, 0x43,
-    0x72, 0xAF, 0x82, 0x1E
+	0x76, 0xf3, 0x14, 0x64, 0x40, 0x98, 0x35, 0x8f, 0x34, 0x47, 0xbd, 0xf7,

```

Comment:
Please remove hard tabs in ca_cert_der_sig[].

Reply:
Replaced tabs with spaces.